beacon-chain/utils/shuffle.go

```diff
@@ -3,12 +3,21 @@ package utils
 
 import (
 	"errors"
+	"fmt"
+	"math"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
 	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
+const hSeedSize = int8(32)
```

Comment:
Please remove the `h` prefix. It is not clear what that means.

Reply:
thanks, done.